src/components/Calculator.js

```diff
@@ -1,106 +1,40 @@
-              %
-            </button>
-            <button type="button" id="ac">
-              AC
-            </button>
-          </div>
-
-          <div className="row">
-            <button type="button">sin</button>
-            <button type="button">π</button>
-            <button type="button" className="btn">
-              7
-            </button>
-            <button type="button" className="btn">
-              8
-            </button>
-            <button type="button" className="btn">
-              9
-            </button>
-            <button type="button" className="btn">
-              ÷
-            </button>
-          </div>
-
-          <div className="row">
-            <button type="button">cos</button>
-            <button type="button">log</button>
-            <button type="button" className="btn">
-              4
-            </button>
-            <button type="button" className="btn">
-              5
-            </button>
-            <button type="button" className="btn">
-              6
-            </button>
-            <button type="button" className="btn">
-              ×
-            </button>
-          </div>
+class Calculator extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.onButtonClick = this.onButtonClick.bind(this);
+    this.state = {
+      obj: {
+        total: null,
+        next: null,
+        operation: null,
+      },
+    };
+  }
 
-          <div className="row">
-            <button type="button">tan</button>
-            <button type="button">√</button>
-            <button type="button" className="btn">
-              1
-            </button>
-            <button type="button" className="btn">
-              2
-            </button>
-            <button type="button" className="btn">
-              3
-            </button>
-            <button type="button" className="btn">
-              -
-            </button>
-          </div>
+  onButtonClick = (e) => {
+    const buttonName = e;
+    const { obj } = this.state;
+    this.setState({
+      obj: calculate(obj, buttonName),
+    });
+  };
 
-          <div className="row">
-            <button type="button">e</button>
-            <button type="button">
-              x
-              <span style={{ position: 'relative', bottom: '.4em', right: '.1em' }}>y</span>
-            </button>
-            <button type="button" className="btn">
-              0
-            </button>
-            <button type="button" className="btn">
-              .
-            </button>
-            <button type="button" id="eval">
-              =
-            </button>
-            <button type="button" className="btn">
-              +
-            </button>
-          </div>
-        </div>
+  render() {
+    const { obj } = this.state;
+    return (
+      // <div className="container">
+      //   <InputResult displayState={obj} />
+      //   <Button buttonClick={this.onButtonClick} />
+      // </div>
```

Comment:
- [ ] Kindly remove commented code because it makes your codebase look a bit messy.